docs_user/assemblies/assembly_migrating-ceph-rbd.adoc

```diff
@@ -0,0 +1,30 @@
+[id="migrating-ceph-rbd_{context}"]
+
+:context: migrating-ceph-rbd
+
+= Migrating Red Hat Ceph Storage RBD
+
+For hyperconverged infrastructure (HCI) or dedicated Storage nodes that are running version 5 or later, you must migrate the daemons that are included in the {rhos_prev_long} control plane into the existing external RHEL nodes. The external RHEL nodes typically include the Compute nodes for an HCI environment or dedicated storage nodes.
```

Comment:
s/version 5/version 6/g

We have no rolled out Ceph 7 with 17, so let's keep `6 or later` instead of 5 (that was a valid choice at the time of writing). 
Also +1 on the second sentence: `HCI = (Compute + CephOSD)`